pkg/config/remote/api/http.go

```diff
@@ -197,3 +199,50 @@ func (c *HTTPClient) FetchOrgData(ctx context.Context) (*pbgo.OrgDataResponse, e
 
 	return response, err
 }
+
+func (c *HTTPClient) FetchOrgStatus(ctx context.Context) (*pbgo.OrgStatusResponse, error) {
+	url := c.baseURL + orgStatusEndpoint
+	log.Debugf("Querying url %s", url)
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, url, &bytes.Buffer{})
+	if err != nil {
+		return nil, fmt.Errorf("failed to create org data request: %w", err)
+	}
+	req.Header = c.header
+
+	resp, err := c.client.Do(req)
+	if err != nil {
+		return nil, fmt.Errorf("failed to issue org data request: %w", err)
+	}
+	defer resp.Body.Close()
+
+	var body []byte
+	// Specific case: authentication method is wrong
+	// we want to be descriptive about what can be done
+	// to fix this as the error is pretty common
+	if resp.StatusCode == 401 {
+		return nil, ErrUnauthorized
+	}
+
+	if resp.StatusCode >= 400 && resp.StatusCode <= 499 {
+		return nil, fmt.Errorf("%w: %d", ErrProxy, resp.StatusCode)
+	}
+
+	// Any other error will have a generic message
+	if resp.StatusCode != 200 {
+		return nil, fmt.Errorf("non-200 response code: %d", resp.StatusCode)
+	}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ Question:** ‏Isn't this used in other places? Should we try to regroup these checks somewhere?